portfolio/src/main/webapp/demographic.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Rania's Portfolio - Demographic</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <script src="apiKey.js"></script>
+    <script src="mapScript.js"></script>
+    <script src="nightVersion.js"></script>
+    <script src="https://unpkg.com/@google/markerclustererplus@4.0.1/dist/markerclustererplus.min.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload=loadingAlert()>
+    <div id="content">
+
+      <!-- navigation bar -->
+      <nav>
+        <a href="index.html" class="bar-item w3-button">Home</a>
+        <a href="experience.html" class="bar-item w3-button">Experience</a>
+        <a href="/files/CV_Rania_Rusly.pdf" class="bar-item w3-button">Resume</a>
+        <a href="interest.html" class="bar-item w3-button">Interest</a>
+        <a href="gallery.html" class="bar-item w3-button">Gallery</a>
+        <a href="demographic.html" class="bar-item w3-button">Demographic</a>
+        <a href="contact.html" class="bar-item w3-button">Contact Me</a>
+        <div class="name-bar-item">Rania Rusly</div>
+      </nav>
+      <div style="height: 40px;"></div>
+
+      <div class="map-section">
+        <h1>Demographic</h1>
+        <p>Hi I am trying out the Geolocation API and the map below is the result!
+           The map shows the location where the page views come from
+           To see your mark on the map, please allow the location enabling, thank you!</p>
+        <br><br>
+        <div id="user-map" class="map"></div>
+      </div>
+
+      <div class="footer">
+        <!-- social media link -->
+        <div class = "sns-container">
+          <a href="https://www.instagram.com/raniatamara/">
+            <i class="sns-icon sns-icon w3-button fa fa-instagram " style="font-size:40px"></i>
```

Comment:
Nit: `sns-icon` is duplicated here.

Also, you could put the style `font-size:40px` in CSS under `.sns-icon` CSS selector.